### PR TITLE
Prac 05 feedback 

### PR DESCRIPTION
**Pull request by JevanChandra**
Please help review my code
@a-rosa
@AdriantoHartono
@Riston-gitting-gud
@MikeLeongZhiFai

### CODE REVIEW

File: cp1404practicals/prac_05/emails.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    name = " ".join(email.split("@")[0].split(".")).title()
+    choice = input(f"Is your name {name}? (Y/n) ").lower()
+    if choice != "" and choice != "y":
+        name = input("Name: ").title()
+    email_dictionary[name] = email
+    email = input("Email: ")
+
+for email in email_dictionary.items():
+    print(email)
+    print(f"{email[0]} ({email[1]})")
```
Comment (MikeLeongZhiFai) [inline cp1404practicals/prac_05/emails.py:21]: Hi bro, i do not know how to do this question tbh, but I used your code as reference. You method helped me a lot!

File: cp1404practicals/prac_05/hex_colours.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    print(f"{COLOUR_TO_CODE[colour]} is {colour}")
+colour_input = input("What colour: ").upper()
+
+while colour_input != "":
+    if colour_input in COLOUR_TO_CODE:
+        print(colour_input, "is", COLOUR_TO_CODE[colour_input])
+    else:
+        print("Invalid colour")
+    colour_input = input("What colour: ").upper()
+
```
Comment (MikeLeongZhiFai) [inline cp1404practicals/prac_05/hex_colours.py:22]: Nice, code is easy to understand

**Review by MikeLeongZhiFai — COMMENTED** [review]
Code is working and easy to understand

**Review by MikeLeongZhiFai — COMMENTED** [review]
This is a hard one, code is easy to understand, took some of it as reference as well!

**Review by MikeLeongZhiFai — COMMENTED** [review]
code is easy to read and understand!

File: cp1404practicals/prac_05/emails.py (hunk 1 above)
Comment (JevanChandra) [reply cp1404practicals/prac_05/emails.py:21]: No worries bro feel free to

File: cp1404practicals/prac_05/emails.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_05/emails.py:1]: Good code and simple, nice 👌

File: cp1404practicals/prac_05/hex_colours.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_05/hex_colours.py:1]: Good code, it can work as expected 👌

File: cp1404practicals/prac_05/state_name.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_05/state_name.py:1]: The code is simple and functionable, good 👍

File: cp1404practicals/prac_05/wimbledon.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_05/wimbledon.py:1]: Nice, the code can give an output as expected from the question 👌

File: cp1404practicals/prac_05/word_occurrences.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_05/word_occurrences.py:1]: Good code and it give an output as expected on the questions 👌

File: cp1404practicals/prac_05/wimbledon.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            country_winners.append(country)
+
+    print("Wimbledon Champions: ")
+    for winner in winners.items():
+        print(f"{winner[0]} {winner[1]}")
+    print()
+
+    country_winners.sort()
+    print(f"These {len(country_winners)} countries have won Wimbledon: ")
+    print(", ".join(country_winners))
```
Comment (a-rosa) [inline cp1404practicals/prac_05/wimbledon.py:30]: Great code, but you need to use functions here

**Pull request merged**